lib/ssl/src/dtls_record.erl

```diff
@@ -271,77 +271,65 @@ decode_cipher_text(#ssl_tls{epoch = Epoch} = CipherText, ConnnectionStates0) ->
 %% or vice versa.
 %%--------------------------------------------------------------------
 protocol_version('dtlsv1.2') ->
-    {254, 253};
+    ?DTLS_1_2;
 protocol_version(dtlsv1) ->
-    {254, 255};
-protocol_version({254, 253}) ->
+    ?DTLS_1_0;
+protocol_version(?DTLS_1_2) ->
     'dtlsv1.2';
-protocol_version({254, 255}) ->
+protocol_version(?DTLS_1_0) ->
     dtlsv1.
```

Comment:
I think it would be better to go for names that does not correspond to the representation! I think we could make it two functions protcol_version_name would convert to the official API names that are atoms, and we can keep protocol_version for the internal representation. I think we should keep protocol in the name so that it is clear that it is that version and not some other version that is handled.

Reply:
I agree that we should not have function names indicating what representation we have so `version_name_to_internal` and `version_internal_to_name` could be one suggestion. I am not sure that `protocol`need to be part of the name since the functions are in `dtls_record` and probably some corresponding tls/ssl module as well.